lib/Doctrine/Common/Annotations/CachedReader.php

```diff
@@ -213,14 +208,13 @@ private function saveToCache($cacheKey, $value)
     /**
      * Checks if the cache is fresh.
      *
-     * @param string           $cacheKey
-     * @param ReflectionClass $class
+     * @param string $cacheKey
      *
      * @return boolean
      */
     private function isCacheFresh($cacheKey, ReflectionClass $class)
     {
-        if (null === $lastModification = $this->getLastModification($class)) {
+        if (0 === $lastModification = $this->getLastModification($class)) {
```

Comment:
Let's remove the assignment from the condition and the yoda expression? Two nasty things in a single line is too much for me 🤣

Reply:
Well I'm not going to be the one who defends Yoda, that's for sure :P https://dev.to/greg0ire/why-using-yoda-conditions-you-should-probably-not